code/game/objects/objs.dm

```diff
@@ -322,6 +322,9 @@
 		if(!do_after(user, (fumble_time ? fumble_time : repair_time) * (skill_required - user.skills.getRating(SKILL_ENGINEER)), NONE, src, BUSY_ICON_BUILD))
 			return TRUE
 
+	if(user.skills.getRating(SKILL_ENGINEER) > SKILL_ENGINEER_ENGI)
+		repair_amount *= (1+(0.1*(user.skills.getRating(SKILL_ENGINEER) - SKILL_ENGINEER_ENGI)))
```

Comment:
why are you referencing the engie level of skill? Ref the skill_required var.

Reply:
Because my intent is to make ENGI (3) as the baseline of 100% repair amount, because how metal and plasteel barricade repair only have skill_required at Metal (1) and Plasteel (2) level of engineer, if i put it at those then it mean mostly Everyone going to have increased repair amount. Or you would prefer it to be like that?

Reply:
What

Reply:
Like this? 
        if(user.skills.getRating(SKILL_ENGINEER) > skill_required) 
                 repair_amount *= (1+(0.1*(user.skills.getRating(SKILL_ENGINEER) - skill_required)

Or this

if(user.skills.getRating(SKILL_ENGINEER) > skill_required) 
                 repair_amount *= (1+(0.1*(user.skills.getRating(SKILL_ENGINEER) - SKILL_ENGINEER_ENGI)

Reply:
This is a kuro question since its balance, but both ways have issues. Arbitrarily choosing SKILL_ENGINEER_ENGI though is specifically problematic if something in the future has a higher requirement. If I need level 5 to repair something for example, its unintuitive to repair it for 120% or whatever just for meeting the requiment.

Reply:
Fair, so skill_required + 1 should be fine then?